include/parser_interface.hpp

```diff
@@ -26,6 +26,19 @@ class ParserInterface
      */
     virtual types::VPDMapVariant parse() = 0;
 
+    /**
+     * @brief Virtual function to perform write on VPD of any type.
+     *
+     * The API can be overridden by classes inheriting this ParserInterface
+     * class to implement VPD write operation.
+     *
+     * @return 0 on successful write, 1 on failure.
```

Comment:
we are still discussing on returning either size or return code (or even both). so if I keep return type as int, it will be flexible to change just the description.

Reply:
Can we finalize on it asap. As it will effect the implementation and till then have a TODO.
In case size is not required, we can't proceed with int as return type.